### ISSUE TITLE
Followup: ensure the `Planner` correctly makes use of the `SwapClaim` fee

### ISSUE THREAD

**Issue by hdevalence** [penumbra-zone/penumbra#1754]
Followup: ensure the `Planner` correctly makes use of the `SwapClaim` fee

Currently, I don't think the `Planner` is aware that the `SwapClaim` has a contribution to the transaction fee. Probably it should set the claim transaction's fee to the prepaid fee, if the fee is unset.

_Originally posted by @hdevalence in https://github.com/penumbra-zone/penumbra/pull/1736#discussion_r1045176489_

**Comment by hdevalence** [comment]
This is blocked on some planner refactoring for which I have not yet written an issue.

**Comment by hdevalence** [comment]
Issue referenced above is https://github.com/penumbra-zone/penumbra/issues/4081

**Comment by TalDerei** [comment]
Closing as this is accounted for in https://github.com/penumbra-zone/penumbra/pull/4312 and follow-up work superseded by https://github.com/penumbra-zone/penumbra/issues/4313
